### ISSUE TITLE
The Huggingface dataset does not match the dataset used in the paper.

### ISSUE THREAD

**Issue by risebeforeshining** [hanghuacs/V2Xum-LLM#6]
In the Hugging Face dataset, the provided resources currently include the video **.npy** files and the dialogue text file **refined_data.jsonl**. However, the **ChatGPT-4 refined dataset** contains a large amount of duplicated data, and there are also dialogue entries without corresponding .npy files.

According to the paper, the dataset is described as:

> “a dataset that contains 30k open-domain videos, partitioned as 25,000 in the training set, 1,000 in the validation set, and 4,000 in the test set.”

At present, only 19,228 samples can be found.

Would it be possible to provide the complete training dataset?
